src/main/java/com/gitrekt/resort/controller/BookingDetailsListItemController.java

```diff
@@ -0,0 +1,85 @@
+package com.gitrekt.resort.controller;
+
+
+import com.gitrekt.resort.model.BookingDetailEntry;
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+
+/**
+ * This class controls the functionality of BookingDetialsListItem.fxml
+ *
+ */
+public class BookingDetailsListItemController implements Initializable {
+    
+    
+    @FXML
+    private Node root;
+    
+    @FXML
+    private Label bookingNumber;
+    
+    @FXML
+    private Label roomNumber;
+    
+    @FXML
+    private Label bookingStart;
+    
+    @FXML
+    private Label bookingEnd;
+    
+    @FXML
+    private Label roomDescription;
+    
+    @FXML
+    private ImageView roomImage;
+    
+    @FXML
+    private Button cancelButton;
+    
+    @FXML
+    private Button billButton;
+    
+    
+    /**
+     * Initializes the controller class.
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        // TODO
+    } 
+    
+    public void setData(BookingDetailEntry bookingData) {
+        roomImage.setImage(bookingData.getRoom().getImage());
+  
+        roomDescription.setText(
+            bookingData.getRoom().getDescription()
+        );
+        bookingNumber.setText(bookingData.getbookingNumber());
+        
```

Comment:
These empty lines here really aren't necessary. Only separate code within a method when the separated sections represent different logical chunks of code. All of these lines do something very similar, meaning the vertical whitespace is not needed.